packages/fxa-settings/src/react-app-env.d.ts

```diff
@@ -3,3 +3,40 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 /// <reference types="react-scripts" />
+
+type Hash<T> = { [key: string]: T };
+
+interface QueryParams {
+  device_id?: string;
+  flow_id?: string;
+  flow_begin_time?: number;
+}
+
+interface AccountData {
```

Comment:
idk how i feel about this one being here, but we can always move it later if we need to

Reply:
yeah I tend to agree. What if I moved this to its own accounts.d.ts in the AccountHOC component?

Reply:
i've got an idea... lemme sketch it up